algobot/graph_helpers.py

```diff
@@ -133,7 +133,7 @@ def add_data_to_plot(gui: QMainWindow, targetGraph: PlotWidget, plotIndex: int,
     plot['plot'].setData(plot['x'], plot['y'])
 
 
-def setup_graph_plots(gui: QMainWindow, graph: PlotWidget, trader: Trader, graph_type: int):
+def setup_graph_plots(gui: QMainWindow, graph: PlotWidget, trader: Trader, graph_type: GraphType):
```

Comment:
Wouldn't this still be an integer though?

Reply:
Probably with MyPy it would complain ;)

At least this is more descriptive

Reply:
Yes, but my only concern is PyCharm complaining..